caikit/runtime/servicers/global_train_servicer.py

```diff
@@ -162,6 +172,15 @@ def Train(self, request, *_, **__) -> TrainingJob:
             log.warning({**log_dict, **e.metadata})
             raise e
 
+        except concurrent.futures.CancelledError as err:
+            log_dict = {
+                "log_code": "<RUN71530128W>",
+                "message": err.message,
+                "error_id": err.id,
+            }
+            log.warning({**log_dict, **e.metadata})
+            raise err
```

Comment:
Do we need to wrap this in `CaikitRuntimeException` to put a grpc error code on it?

Reply:
actually we don't need this anymore. I'll remove it